CMakeLists.txt

```diff
@@ -43,9 +43,12 @@ set(RGM_SOURCES
     Plugins/RGMPlugin.cpp
     Plugins/ServerPlugin.cpp
     Components/RecentFiles.cpp
-    qtplugins.cpp
 )
 
+if (MSVC)
+  set(RGM_SOURCES ${RGM_SOURCES} qtplugins.cpp)
+endif()
```

Comment:
I have the same question here as I did on the `enigma-dev` pr. Why don't we make our own static flag so that behavior is not dependent on the compiler?